### PR TITLE
fix license and "clear is not a function" error

### PR DESCRIPTION
**Pull request by jlmakes**
Hey Anton,

Thanks for helping bring ScrollReveal to the React community!

### Fixes / Improvements

I noticed an erroneous use of `sr.clear()` in the source, which I presumed to be a typo of the `clean()` method. Also, I upgraded ScrollReveal to the latest version (`4.0.0-beta.23`), which includes [quite a few improvements](https://github.com/jlmakes/scrollreveal/compare/v4.0.0-beta.9...development).

### License

Because `react-scrollreveal` is based off ScrollReveal 4 (which uses the GPLv3), you must—[under the terms of the GPLv3](https://www.gnu.org/licenses/gpl-3.0.html#section5)—distribute this package under the same license.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -106,9 +106,7 @@ const ReactScrollreveal = (srOptions = {}) => (Component) => {
       // clearing styles makes sr animation initialize again
       // on same element that were still in DOM
       if (clearStyles) {
-        this.forEachSrElement((el) => {
-          sr.clear(el);
-        });
+        this.forEachSrElement(sr.clean);
```
Comment (jlmakes) [inline src/index.js]: This only works because `this` is bound to instance methods as of `beta.23`

**Comment by RusinovAnton** [comment]
@jlmakes thanks for your contribution, greatly appreciated!  
I might use a suggestion on version bump. Whats your thoughts on releasing these changes under `1.1.0` which is minor bump?

**Comment by jlmakes** [comment]
Normally, I'd suggest a license change as a major bump—but in this case the current license is invalid, so minor (or even patch) seems preferable.

Also, are you open to other changes? I have some different ideas on how ScrollReveal could be integrated with react; I'm curious to hear what you think of this implementation:

https://codesandbox.io/s/z2wqk3vm1l

**Comment by jlmakes** [comment]
Normally, I'd suggest a license change as a major bump—but in this case the current license is invalid, so minor (or even patch) seems preferable.

**Pull request merged**